bin/push-dir.js

```diff
@@ -2,5 +2,31 @@
 
 var pushDir = require('../index.js');
 
-var opts = require('minimist')(process.argv.slice(2));
-pushDir(opts);
+
+var opts = {
+  boolean: [
+    'preserve-local-temp-branch',
+    'help',
+  ],
+  alias: {
+    'help': 'h',
+  },
+};
+var args = require('minimist')(process.argv.slice(2), opts);
+
+var al = args._.length;
+if (args.help) showUsage(0);
+if (!al || al > 2) showUsage(1);
+
+pushDir(args);
+
+
+function showUsage(exitCode) {
+  var fs = require('fs');
+  var path = require('path');
+  var readme = path.join(__dirname, '..', 'README.md');
+  var readmeText = String(fs.readFileSync(readme));
+  var usage = /```usage([\s\S]*?)```/.exec(readmeText)[1].trim();
```

Comment:
Lol, this is clever but is it worth it for DRYness? How often will usage change in the future once this is API stable. Runtime parsing of the README seems kinda crazy.